collectors/o365/test/o365_test.js

```diff
@@ -401,6 +401,27 @@ describe('O365 Collector Tests', function() {
             });
         });
 
+        it('check if state moment is null or blank', function (done) {
+            setO365MangementStub();
+            O365Collector.load().then(function (creds) {
+                var collector = new O365Collector(ctx, creds, 'o365');
+                const curState = {
+                    stream: "FakeStream",
+                    since: "",
+                    until: null,
```

Comment:
I would add a test case where both since and until are undefined

Reply:
done